aws/resource_aws_ecs_service.go

```diff
@@ -106,12 +106,6 @@ func resourceAwsEcsService() *schema.Resource {
 				Type:     schema.TypeInt,
 				Optional: true,
 				Default:  100,
```

Comment:
Instead of leaving a `Default: 100` here that only works for one of the scheduling strategies, should we instead remove `Default: 100` and implement a `DiffSuppressFunc` that covers suppressing the difference depending on the strategy? A similar update should likely occur for maximum as well.

Reply:
If we remove the `Default` wouldn't that be backwards incompatible for those who are using a `replica` strategy and didn't set `deployment_minimum_healthy_percent`?

Similar for `deployment_maximum_percent`, if we remove `Default` won't that be a problem for backwards compatibility?

Reply:
Ok. So when you say `we allow two defaults chosen by the API`, do you mean by the AWS API? If so then for `deployment_maximum_percent` and `deployment_minimum_healthy_percent` we should set these as `Optional`?